js/ssai_pulse.js

```diff
@@ -381,42 +415,53 @@ OO.Ads.manager(function(_, $)
     this.onVideoTagFound = function(eventName, videoId, tagType, metadata)
     {
       OO.log("TAG FOUND w/ args: ", arguments);
-      this.currentId3Object = _parseId3Object(metadata);
-      if (this.currentId3Object)
+      if(!amc.isLiveStream && !currentOffset)
```

Comment:
@ipadilla4 why was this block added back here? This effectively ignores ID3 tags for VOD, was there a reason to add this back to the code?